lib/ChGovUk/Bridge/Company.pm

```diff
@@ -128,6 +131,53 @@ sub company {
     return undef;
 }
 
+sub get_basket_link {
+    my ( $self ) = @_;
+        if ($self->is_signed_in) {
+        $self->ch_api->basket->get->on(
+            success        => sub {
+                my ($api, $tx) = @_;
+                my $json = $tx->success->json;
+                my $show_basket_link = $json->{data}{enrolled} || undef;
+                my $items = scalar @{$json->{data}{items} || []};
+                if ($show_basket_link) {
+                    debug "User [%s] enrolled for multi-item basket; displaying basket link", $self->user_id, [COMPANY_BRIDGE];
+                }
+                else {
+                    debug "User [%s] not enrolled for multi-item basket; not displaying basket link", $self->user_id, [COMPANY_BRIDGE];
+                }
+                $self->stash_basket_link($show_basket_link, $items);
+            },
+            not_authorised => sub {
+                my ($api, $tx) = @_;
+                debug "User not authenticated; not displaying basket link", [COMPANY_BRIDGE];
+                $self->stash_basket_link(undef, 0);
+            },
+            failure        => sub {
+                my ($api, $tx) = @_;
+                debug "Error returned by getBasketLinks endpoint; not displaying basket link", [COMPANY_BRIDGE];
```

Comment:
You could perhaps log a little more about the underlying error in case someone wants to know why the basket link not showing up?

Reply:
Good idea, will do that now.